### DISCUSSION TITLE
Why isn't the default chart version in the Helm chart repository increasing?

### DISCUSSION THREAD

**Discussion by civilizeddev** [crossplane/crossplane#5302] [Q&A] [answered]
Until recently, the helm chart has been updating,

but why is the default version fixed at `1.15.0-rc.0.9.ged19cd5d`?

https://artifacthub.io/packages/helm/crossplane/crossplane

**Comment by jbw976** [comment]
That's a very good question. Artifacthub seems to know about all the latest Crossplane helm charts published to `master` channel since that version: https://artifacthub.io/api/v1/packages/helm/crossplane/crossplane/feed/rss

And going to one of those newer versions specifically shows a new/correct application version: https://artifacthub.io/packages/helm/crossplane/crossplane/1.15.0-rc.0.363.g22e8adef

Asking the `helm` command line shows a new/updated chart and app version:

```
❯ helm search repo crossplane-master/crossplane --devel
NAME                                    	CHART VERSION            	APP VERSION              	DESCRIPTION
crossplane-master/crossplane            	1.15.0-rc.0.363.g22e8adef	1.15.0-rc.0.363.g22e8adef	Crossplane is an open source Kubernetes add-on ...
```

Looking at the Crossplane [history in master](https://github.com/crossplane/crossplane/commits/master/?after=22e8adef5f3109c218eb9d3a1e8540ef21d903e3+320):
* PR corresponding to the last application version shown in artifacthub: https://github.com/crossplane/crossplane/pull/4919
* Very next merged PR, that one could suspect of breaking it, doesn't even touch the Helm chart: https://github.com/crossplane/crossplane/pull/4910

So I have no idea why Artifacthub is showing an older default version on https://artifacthub.io/packages/helm/crossplane/crossplane/, could this be an Artifacthub bug?  🤔

**Reply by civilizeddev** [reply]
https://github.com/artifacthub/hub/discussions/3633#discussioncomment-8302192

Turns out it's a bug.

**Reply by civilizeddev** [reply, marked as answer]
fixed.

https://github.com/artifacthub/hub/discussions/3633#discussioncomment-8331294

<img width="1792" alt="Screenshot 2024-02-01 at 16 34 44" src="https://github.com/crossplane/crossplane/assets/15084045/90dec2ff-066f-46d4-b7ee-f070eaa775b7">

**Reply by jbw976** [reply]
Amazing @civilizeddev, thank you very much for the thorough follow-up here and providing this update!! 🎉
